sdk/metricsadvisor/azure-ai-metricsadvisor/azure/ai/metricsadvisor/_metrics_advisor_administration_client.py

```diff
@@ -607,12 +608,11 @@ def refresh_data_feed_ingestion(
         )
 
     @distributed_trace
-    def delete_alert_configuration(self, alert_configuration_id, **kwargs):
-        # type: (str, Any) -> None
+    def delete_alert_configuration(self, *alert_configuration_id, **kwargs):
```

Comment:
No. We only accept one argument which is ID. The point here is we want id to be position only argument.

Reply:
okay, new question 😄 - for my curiosity, why do we need it to be positional only?

Reply:
(Was added in a wrong place :()

This is the feedback from arch board review. We want user to just call delete(str) and not need to care about the exact arg name which is xxx_id. This can also help to prevent breaking changes if we want to change the arg name.

Reply:
got it - thanks!